src/components/feedback/categories.js

```diff
@@ -0,0 +1,81 @@
+/* eslint es/no-computed-properties: off */
+
+import React from 'react';
+import CategoryLike from './components/category-like.js';
+import CategoryProblem from './components/category-problem.js';
+import CategoryConfusing from './components/category-confusing.js';
+
+// This function holds the majority of text for the Feedback component
+// It returns an object:
+// - The key is the category's name
+// - `helpful` infers if the user found the content helpful
+// - `component` returns the component for the category
+// To add, remove, or change the categories, update the object and the Feedback will automatically update with your changes.
+export function categories({ type, submitFeedback }) {
+  const genericType = returnGenericType(type);
```

Comment:
This will be variable, you can see this snapshot to see how the content will appear for different content types: https://github.com/mapbox/dr-ui/pull/462/files#diff-10c599eb89dae61a0ebabeefaab0ac598765b263a076a5ea08d0c9893507c3d7

> If yes, will it be difficult to report on trends in mode reports?

This is a good thing to call out, we could:

1. Adjust the value we send to Segment/Sentry to "Easy to understand" while keeping the label variable `The ${genericType} is easy to understand`.
2. Change the label to `It is easy to understand` to remove the variable content type.

What do you think?

Reply:
I like the simplicity/consistency of 2, but I think the specificity of 1 is helpful to guide the users to provide clear feedback. I lean slightly toward 1, but could easily be convinced otherwise if you feel differently!

Reply:
I'll scope out 1!

Reply:
Option 1  was a fairly easy thing to implement 😅  c075e16!